tests/test_irbempy.py

```diff
@@ -290,25 +290,132 @@ def test_GSM_SM_init(self):
 
     def test_get_AEP8(self):
         """test get_AEP8"""
-        c=self.loci
+        c = self.loci
         c.ticks = self.ticks
-        E = 2.0 # energy in MeV
+        E = 2.0  # energy in MeV
         expected = 99492.059080021136
         actual = ib.get_AEP8(E, c)
         numpy.testing.assert_almost_equal(expected, actual)
-# -----------------------------------------------------------------------
-
 
-if	__name__	==	"__main__":
-    ##	suite	=	unittest.TestLoader().loadTestsFromTestCase(SimpleFunctionTests)
-    ##	unittest.TextTestRunner(verbosity=2).run(suite)
 
-    ##	suite	=	unittest.TestLoader().loadTestsFromTestCase(tFunctionTests)
-    ##	unittest.TextTestRunner(verbosity=2).run(suite)
+class IRBEMShieldoseTests(unittest.TestCase):
```

Comment:
Coverage is good, except for repr and str. Should we put in quick tests for those (pending, of course, the resolution of the discussion above).

Reply:
I can. It helps the coverage although I'm not sure that it makes a material difference to functionality. Anyone relying on repr or str instead of querying the object is doing something stupid.